service/schema/AttributeTypeState.kt

```diff
@@ -64,6 +65,8 @@ class AttributeTypeState internal constructor(
     val ownerTypes get() = ownerTypeProperties.map { it.ownerType }
     val ownerTypesExplicit get() = ownerTypeProperties.filter { !it.isInherited }.map { it.ownerType }
 
+    private val loadedOwnerTypePropsAtomic = AtomicBooleanState(false)
```

Comment:
Oh, why did we end up using `AtomicBooleanState`? I thought we didn't gain anything from having a `MutableState` here.

Reply:
This was left over from experimentation. Fixed.